client/src/screens/burger/Index.js

```diff
@@ -0,0 +1,175 @@
+      mutate({
+        variables: { user: { displayName } },
+      }),
+  }),
+});
+
+class Burger extends Component {
+  static navigationOptions = {
+    title: 'More',
+    tabBarIcon: ({ tintColor }) => <Icon size={28} name="bars" color={tintColor} />,
+  };
+
+  updateLocation = () => {
+    const reportError = (error) => {
+      Alert.alert('Error updating location', error.message);
+    };
+
+    // `navigator` is a browser polyfill and does not need to be imported
+    navigator.geolocation.getCurrentPosition(
+      (position) => {
+        const { latitude, longitude } = position.coords;
+        this.props
+          .updateLocation({ locationLat: latitude, locationLon: longitude })
+          .then((result) => {
+            const success = result.data.updateLocation;
+            console.log(`Updated location: ${latitude},${longitude} (${success})`);
+          })
+          .catch((err) => {
+            console.log(`Failed to update location: ${latitude},${longitude} (${err})`);
+            reportError(err);
+          });
+      },
+      (err) => {
+        reportError(err);
+      },
+      { enableHighAccuracy: true, timeout: 20000, maximumAge: 1000 },
+    );
+  }
+
+  showEventResponse = () => {
+    const { navigate } = this.props.navigation;
+    navigate('EventResponse');
+  }
+
+  showUserProfile = () => {
+    const { navigate } = this.props.navigation;
+    navigate('Profile');
+  }
+
+  logout = () => {
+    this.props.dispatch(logout());
+  }
+
+  about = () => {
+    const DeviceInfoAvailable = DeviceInfo.typeof;
+
+    codePush.getCurrentPackage().then((result) => {
+      Alert.alert(
+        'About',
+        `
+        getBundleId: ${DeviceInfoAvailable ? DeviceInfo.getBundleId() : 'NA'}
+        BuildNumber: ${DeviceInfoAvailable ? DeviceInfo.getBuildNumber() : 'NA'}
+        getReadableVersion: ${DeviceInfoAvailable ? DeviceInfo.getReadableVersion() : 'NA'}
+        CodePushlabel: ${result ? result.label : 'No CP package installed'}
+        CodePushHash: ${result ? result.packageHash : 'No CP package installed'}
+        `,
+      );
+    });
+  }
+
+    checkForUpdate = () => {
+      codePush.checkForUpdate()
+        .then((update) => {
+          if (update) {
+            Alert.alert(
+              'Update Available',
+              `Version ${update.appVersion} (${(update.packageSize / 1024 / 1024).toFixed(2)}mb) is available for download`,
+              [
+                { text: 'Cancel', style: 'cancel' },
+                { text: 'Install & Restart', onPress: this.installUpdate },
+              ],
+              { cancelable: false },
+            );
+          } else {
+            Alert.alert('No updates available.');
+          }
+        });
+    }
+
+    installUpdate = () => {
+      codePush.sync({ updateDialog: false, installMode: codePush.InstallMode.IMMEDIATE });
+    }
+
+    render() {
+      return (
+        <Center>
+          <Text />
```

Comment:
i really hate this is padded with `<Text />` but I'll take it for now.

Reply:
Yea, i just left that in place. Thats carry over from original code.

Re Buttons - they are components. I reused Scotts component.

import { Button } from '../../components/Button';